src/process.rs

```diff
@@ -157,16 +158,20 @@ fn push_possible_output(
     if stdout.is_some() || stderr.is_some() {
         let mut poll_lock = poll_data.lock().unwrap();
         if let Some(dat) = stdout {
-            poll_lock.push(PollData {
-                typ: PollType::Stdout,
-                data: dat,
-            });
+            if !dat.is_empty() {
```

Comment:
That's not how Some() works lol.... right?

Reply:
It's always Some(), its only None if there is no pipe

```
Communicate with the subprocess, return the contents of its standard output and error.
This will write input data to the subprocess's standard input and simultaneously read its standard output and error. The output and error contents are returned as a pair of Option<Vec>. The None options correspond to streams not specified as Redirection::Pipe when creating the subprocess.
```

Reply:
change is_some to is_empty then above a few lines

Reply:
Resolved, maybe? Not entirely sure what you want